services/brig/src/Brig/Data/User.hs

```diff
@@ -128,9 +129,10 @@ newAccount u inv tid = do
     pict = fromMaybe noPict (newUserPict u)
     assets = newUserAssets u
     status =
-      if isNewUserEphemeral u
-        then Ephemeral
-        else Active
+      if
+          | isNewUserEphemeral u -> Ephemeral
+          | isNewUserInvitedViaScim u -> PendingInvitation
```

Comment:
Is your intention here to remove `isNewUserInvitedViaScim` from `Wire.API.User`? Because it is called from  `validateNewUserPublic` in `Wire.API.User`.

Reply:
right.  i'll take it back, then, it would only have made sense if this was the only call to that function.